frontend/dispatcher.lua

```diff
@@ -801,7 +801,11 @@ function Dispatcher:addSubMenu(caller, menu, location, settings)
             return location[settings] ~= nil and next(location[settings]) == nil
         end,
         callback = function(touchmenu_instance)
+            local name = location[settings].settings and location[settings].settings.name
             location[settings] = {}
+            if name then
```

Comment:
The `checked_func` needs to be updated to match. (This is why i did not want to add a settings item to the table)

Reply:
It was you idea to add 'name' to settings (I liked it).

Reply:
> The `checked_func` needs to be updated to match.

For sure, after https://github.com/koreader/koreader/pull/9651#discussion_r1000918149.